content/teams/2.community.md

```diff
@@ -119,7 +118,7 @@ members:
     location: 'Rennes, France'
```

Comment:
Can you change the location please Clem lives in Bordeaux ;-)
Also Yaël lives in Nantes